### DISCUSSION TITLE
don't show the url text when preview data is available

### DISCUSSION THREAD

**Discussion by m-j-g** [flyerhq/flutter_chat_ui#89] [Ideas & Requests]
It's odd to have the link displayed above the preview data when it is a link to a website, but it's way weirder when it's an image or a gif.  Users just want to see the site preview data, the gif, or the image.  In the case of the gif it makes the user think they should click on the link, when the intention was just to show the gif.

**Comment by demchenkoalex** [comment]
Not critical. Right now a link preview is only available for `TextMessage`, if we will detect that content is only an image or a gif, we will need to somehow convert that to `ImageMesage` which will probably result in strange animations and flickering. This will require re-writing the library once more, which is not planned ATM.
